app/src/main/java/com/leoart/koreanphrasebook/data/repository/AlphabetRepository.kt

```
@@ -35,9 +37,17 @@ class AlphabetRepository(private val context: Context) : CachedRepository<ELette
     }
 
     override fun saveIntoDB(items: List<ELetter>) {
-        localDB().subscribe {
-            it.letterDao().insertAll(*items.toTypedArray())
-        }
+        isEmpty().observeOn(Schedulers.io())
+                .subscribe({
+                    if (it.isSyncNeeded) {
+                        localDB().subscribe { db ->
+                            db.letterDao().insertAll(*items.toTypedArray())
+                            DataInfoRepository.getInstance().updateSyncInfo(SyncModel(ELetter::class.java.simpleName, false))
+                        }
+                    }
+                }, {
+                    it.printStackTrace()
```

Why error is not handled?